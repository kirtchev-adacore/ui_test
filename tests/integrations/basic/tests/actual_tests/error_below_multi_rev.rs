//@no-rustfix
//@revisions: edition2015 edition2018
//@[edition2015] edition:2015
//@[edition2018] edition:2018

#![deny(dead_code, unused_imports)]

//~v unused_imports
use ::std::vec;

mod m {
  //~[edition2015]v dead_code
  pub fn foo() {}
}

//~[edition2015]| unused_imports
//~[edition2018]v E0432
use ::m::foo;

fn main() {}
