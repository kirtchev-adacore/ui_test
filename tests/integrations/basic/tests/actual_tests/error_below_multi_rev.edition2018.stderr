error[E0432]: unresolved import `m`
  --> tests/actual_tests/error_below_multi_rev.rs:18:7
   |
18 | use ::m::foo;
   |       ^ help: a similar path exists: `self::m`

error: unused import: `::std::vec`
 --> tests/actual_tests/error_below_multi_rev.rs:9:5
  |
9 | use ::std::vec;
  |     ^^^^^^^^^^
  |
note: the lint level is defined here
 --> tests/actual_tests/error_below_multi_rev.rs:6:20
  |
6 | #![deny(dead_code, unused_imports)]
  |                    ^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.
